cpu/saml21/include/periph_cpu.h

```diff
@@ -73,6 +73,20 @@ typedef enum {
  */
 #define DAC_NUMOF           (2)
 
+/**
+ * @name    Real time counter configuration
+ * @{
+ */
+#define RTT_DEV             RTC->MODE0
+#define RTT_IRQ             RTC_IRQn
+#define RTT_ISR             isr_rtc
```

Comment:
those are unused

Reply:
`RTT_IRQ` is used, I can remove the other two if you want. As well as `RTC_DEV`

Reply:
I can't find a use of `RTT_IRQ` inside `cpu/sam0_common`

Reply:
Master was using RTC_IRQ https://github.com/RIOT-OS/RIOT/blob/beaec4c2ed906339e4cf3ea8b694bad6c920bb0b/cpu/sam0_common/periph/rtt.c#L127 I changed it to RTT_IRQ, so you would only find the use in this PR.

Reply:
But `RTC_IRQn` is the correct interrupt from `IRQn_Type`.

Reply:
Ok. I'll remove.

Reply:
Removed!